cfgov/scripts/http_smoke_test.py

```diff
@@ -29,7 +29,7 @@
 )
 
 FULL = False
-TIMEOUT = 10
+TIMEOUT = 30
```

Comment:
It might be worth making this a command-line argument so that the timeout could be passed in via Jenkins. However, not necessary to change in this PR.